packages/itwinui-react/src/core/Dialog/DialogMain.tsx

```diff
@@ -207,7 +208,6 @@ export const DialogMain = React.forwardRef<HTMLDivElement, DialogMainProps>(
         classNames={{
           enter: 'iui-dialog-animation-enter',
           enterActive: 'iui-dialog-animation-enter-active',
-          enterDone: 'iui-dialog-visible',
```

Comment:
Removed this classname because CSSTransition classnames were not working properly (It worked in storybook but not in playground). I think it requires to have `*-enter-done` at the end of the classname.

Reply:
Can you move all this into a separate PR? this PR is for documentation only

Reply:
#1197